tox.ini

```diff
@@ -15,7 +15,7 @@ skip_missing_interpreters = true
 
 [testenv]
 description = run tests
-passenv = TOXENV CI
+passenv = TOXENVCI
```

Comment:
Tox v4 was a complete rewrite: https://tox.wiki/en/latest/changelog.html#v4-0-0a1
I can't find any mention of TOXENV in the v4 docs like there were for v3. We might be able to completely remove this line.

Reply:
Just re-ran the pipeline without that line and it still completed, good catch